apps/web/src/app/raidpool/_component/daily-gambit.tsx

```
@@ -0,0 +1,100 @@
+
+                <TooltipContent
+                  side="bottom"
+                  className="max-w-xs bg-background border border-border p-3"
+                >
+                  <p className="text-xs font-medium mb-1">{gambit.name}</p>
+                  <div className="space-y-1">
+                    {gambit.description.map((line, i) => (
+                      <p
+                        key={i}
+                        className="text-[11px] leading-snug"
+                        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)